cmd/harborAPI.go

```diff
@@ -128,35 +149,56 @@ func update(token string, url string, data interface{}) {
 	if Verbose {
 		log.Println(body)
 	}
+
+	return res, body, err
+}
+
+func delete(token string, url string) (*http.Response, string, []error) {
+
+	res, body, err := gorequest.New().
+		Delete(url).
+		Set("x-username", User).
+		Set("x-token", token).
+		End()
+
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	if Verbose {
+		log.Println(body)
+	}
+
+	return res, body, err
 }
 
 // GetLogs returns a string of all container logs for a shipment
 func GetLogs(barge string, shipment string, env string) string {
-		values := make(map[string]interface{})
-		values["barge"] = barge
-		values["shipment"] = shipment
-		values["env"] = env
-    template, _ := uritemplates.Parse(helmitURI + "/harbor/{barge}/{shipment}/{env}")
-		uri, _ := template.Expand(values)
-
-		_, body, err := gorequest.New().
-			Get(uri).
-			End()
-
-		if err != nil {
-			log.Fatal(err)
-		}
+	values := make(map[string]interface{})
```

Comment:
looks like I need to get on the same page with you when it comes to formatting. This is go fmt doing this as a plugin in VSCode?